torch/csrc/jit/runtime/register_backend_ops.cpp

```diff
@@ -0,0 +1,77 @@
+#include <ATen/core/stack.h>
+#include <torch/csrc/jit/backends/backend.h>
+#include <torch/csrc/jit/runtime/custom_operator.h>
+#include <torch/custom_class_detail.h>
+
+namespace torch {
+namespace jit {
+namespace {
+c10::AliasAnalysisKind aliasAnalysisFromSchema() {
+  return c10::AliasAnalysisKind::FROM_SCHEMA;
+}
+
+c10::AliasAnalysisKind aliasAnalysisSpecialCase() {
+  return c10::AliasAnalysisKind::INTERNAL_SPECIAL_CASE;
+}
+
+// Register operators that allow for calling into the custom compile and execute
+// methods for a JIT backend.
+RegisterOperators reg({
+    Operator(
+        "prim::backend_create(str a, Any a) -> Capsule",
+        [](Stack& stack) {
+          auto module = pop(stack);
+          c10::QualifiedName qual_name(pop(stack).toStringRef());
+          // Look up the registered creator function for backend which makes an
+          // instance of the backend that knows how to compile and execute for
+          // it.
+          auto* create_fn = getCustomClassMethod(qual_name);
+          TORCH_CHECK(
+              create_fn,
+              "Backend ",
+              qual_name.name(),
+              " not found; did you forget to register it?");
+          // Run that function to create a backend instance and return it.
+          push(stack, module);
+          create_fn->run(stack);
+          return 0;
+        },
+        aliasAnalysisFromSchema()),
+    Operator(
+        "prim::backend_compile(Capsule c, Any a, Dict(str, Any) b) -> Dict(str, Any)",
+        [](Stack& stack) {
+          auto method_compile_spec = pop(stack).toGenericDict();
+          auto module = pop(stack);
+          auto capsule = pop(stack).toCapsule();
+          auto backend = at::dynamic_intrusive_pointer_cast<
+              PyTorchBackendInterface,
+              CustomClassHolder>(capsule);
+          TORCH_INTERNAL_ASSERT(backend);
+          auto res = backend->compile(module, method_compile_spec);
+          push(stack, res);
+          return 0;
+        },
+        aliasAnalysisFromSchema()),
+    Operator(
+        "prim::backend_execute(Capsule c, Any a, ...) -> Any",
```

Comment:
If you register this op as `prim::backend_execute(Capsule c, Any a, ...) -> ...`, it is not possible to call from Python. If that's what we want to do, I think some changes to how ops are located and dispatched to might be necessary.